src/widgets/number_input.rs

```diff
@@ -503,15 +503,15 @@ where
                             if new_val.is_empty() {
                                 new_val = T::zero().to_string();
                             }
-
+                            
                             match T::from_str(&new_val) {
                                 Ok(val)
-                                    if (self.min..self.max).contains(&val) && val != self.value =>
```

Comment:
Oh i see what you mean now, I thought you meant change how number input takes in the range. Sorry. 
Yeah the ..= would have worked fine here as a range bound lookup. However I think the if val >= self.min && val <= self.max => { is probably a faster method of dealing with it.